packages/@divvi/mobile/src/send/SendConfirmation.test.tsx

```diff
@@ -66,10 +58,25 @@ const mockPrepareTransactionsResultPossible: PreparedTransactionsPossible = {
   feeCurrency: mockFeeCurrencies[0],
 }
 
-type ScreenProps = NativeStackScreenProps<
-  StackParamList,
-  Screens.SendConfirmation | Screens.SendConfirmationFromExternal
->
+const mockBaseScreenProps = {
+  transactionData: {
+    ...mockTokenTransactionData,
+  },
+  origin: SendOrigin.AppSendFlow,
+  isFromScan: false,
+}
+
+const mockSendConfirmationProps = getMockStackScreenProps(Screens.SendConfirmation, {
+  ...mockBaseScreenProps,
+  prepareTransactionsResult: mockPrepareTransactionsResultPossible,
+})
+
+const mockSendConfirmationFromExternalProps = getMockStackScreenProps(
```

Comment:
```suggestion
const mockSendConfirmationFromDeepLinkProps = getMockStackScreenProps(
```
could perhaps move this to the scope of the deep link group (inside the `describe`)?